hermes-mock/src/main/java/pl/allegro/tech/hermes/mock/Request.java

```diff
@@ -0,0 +1,39 @@
+package pl.allegro.tech.hermes.mock;
+
+import com.github.tomakehurst.wiremock.http.RequestMethod;
+import com.github.tomakehurst.wiremock.verification.LoggedRequest;
+
+import java.util.Map;
+
+import static java.util.stream.Collectors.toMap;
+
+public class Request {
+    private String url;
+    private RequestMethod method;
```

Comment:
If we dont' want to expose wiremock to user, we should create our enum`RequestMethod`